src/features/Students/data/api.js

```diff
@@ -75,11 +75,33 @@ function getStudentsByEmail(studentEmail, options = {}) {
   );
 }
 
+/**
+ * Retrieves voucher information filtered by master course ID.
+ *
+ * Sends a GET request to the institution vouchers endpoint, including the
+ * provided master course ID as a query parameter.
+ *
+ * @param {string} examSeriesCode - The exam series code to filter vouchers by.
+ *
+ * @returns {Promise<AxiosResponse>} A promise resolving to the HTTP response object
+ */
+function getInstitutionVouchersByCourse(examSeriesCode) {
+  const params = {
+    exam_series_code: examSeriesCode,
+  };
```

Comment:
@01001110J I think we should update this to `getInstitutionVouchersByExamSeriesCode`, also update the docstring to reflect this change